clients/tabby-agent/src/AgentConfig.ts

```diff
@@ -10,6 +10,9 @@ export type AgentConfig = {
       experimentalStripAutoClosingCharacters: boolean;
       maxPrefixLines: number;
       maxSuffixLines: number;
+      filepath: {
+        experimentalEnabled: boolean;
```

Comment:
i feel this is not necessary since the content should be always filled as long as agent can detect it.